src/main/resources/org/jenkinsci/plugins/workflow/steps/PushdStep/help-path.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    The relative path of the directory to the workspace directory.
```

Comment:
I am not sure on this one, I looked at the Java class here https://github.com/jenkinsci/workflow-basic-steps-plugin/blob/474cea2a53753e1fb9b166fa1ca0f6184b5cee4a/src/main/java/org/jenkinsci/plugins/workflow/steps/PushdStep.java#L38
the comment there is `Temporarily changes the working directory.`.

I think it's more to change the folder to work in.

I do a suggestion but I am not sure on my English :), feel free to change it.

```suggestion
    The relative path in workspace to the folder to use as a new working directory.
```

Reply:
Thank you @aHenryJard. What about this: The relative path of the folder to the workspace to use as a new working directory?

Reply:
Cynthia, I like your rewrite, but how about going back to "directory" rather than "folder"?  I know that the terms are generally used to be synonymous but Jenkins does have a folders feature so it could be ambiguous.  Also, how about "directory in the workspace"?

Reply:
Okay, thank you @Stackscribe. I have made the changes.